src/style.css

```diff
@@ -0,0 +1,336 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+body {
+  font-family: 'Courgette', cursive, monospace, sans-serif, serif;
+  overflow-x: hidden;
+  margin: 0;
+  min-height: 100vh;
+}
+
+.max-w {
+  max-width: 1200px;
+  margin: auto;
+}
+
+header {
+  padding: 0 10px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+}
+
+.logo {
+  width: 100px;
+  height: auto;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  gap: 50px;
+}
+
+.nav-link {
+  list-style-type: none;
+}
+
+.kb-hero-b {
+  width: 100vw;
+  height: 35vh;
+  background-image: url("../assets/kb-hero.png");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] While it is fantastic to be creative and colourful, I genuinely commend you for that, kindly remove this image as it does not match the given [mockup design](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/images/Home.png). For this project, you are expected to adhere to the given design in terms of the data displayed on the page, you can only personalize the colours, typographies, and spacings.

Reply:
Understood, thanks 👍🏽